### DISCUSSION TITLE
View Options button in Project board not working - has anyone else experienced this? How do I fix this?

### DISCUSSION THREAD

**Discussion by coletteocampo-os** [community/community#123042] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Hello! I am suddenly unable to open the View Option for any project board. My account is the only one affected in my organization, it appears. Has anyone else experienced this, or knows of any steps I can take to fix it? Any help would be much appreciated. 

User objective: Export view to TSV
Issue: "View options" dropdown button not working
<img width="189" alt="Screenshot 2024-05-09 at 9 52 58 AM" src="https://github.com/community/community/assets/139727159/95e8df4b-350e-4420-8ddb-cf120a9c70e2">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by bohanwood** [comment]
Same issue here. It's serious!!

**Comment by coletteocampo-os** [comment]
Upon retrying a minute ago, the "View Options" button now works again for me. 🎉

**Reply by ghost** [reply]
It seems it has been fixed!
